tests/mocha/test.js

```diff
@@ -32,6 +32,15 @@ describe('Mocha smoke test', () => {
         assert.equal(el.$('.selector-2').isExisting(), true)
     })
 
+    it('should handle promises in waitUntil callback funciton', () => {
+        const results = []
+        browser.waitUntil(() => {
```

Comment:
This looks good to me, one more note: let's do 
```suggestion
        const result = browser.waitUntil(() => {
```
and make an assertion on the result: `expect(result).toBe(true)`

Reply:
nice catch, missed that

Reply:
@christian-bromann done.